public/app/modules/home/home.controller.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by tremaine on 1/15/16.
+ */
+(function(){
+    'use strict';
+    angular
+        .module('home')
+        .controller('Home', Home);
+
+    Home.$inject = ['$scope', 'dataService'];
+    /**
+     *
+     * @constructor
+     */
+    function Home($scope, dataService){
+        $scope.interval = 5000;
+        $scope.noWrapSlides = false;
+
+        activate();
+
+        function activate(){
+            dataService.getPosts().then(function(data){
+                console.log(data);
+            });
+        }
+
+        $scope.slides = [{
```

Comment:
@tremainebuchanan please change this from being hardcoded to something configurable.  Doesn't have to get done right away, but remember the importance of it.  Open an issue with this.